inventory/group_vars/local.yml

```diff
@@ -18,3 +18,21 @@ developer_email: admin@example.com
 
 users_sysadmin:
   - "{{ core_devs }}"
+
+# Test host configuration
```

Comment:
Seeing as the whole section below is pertaining to test/local hosts, maybe we can make this look more like section comment:

```suggestion
# *** Test host configuration ***
```
(I'm not sure if we have another convention in this codebase already, or you have a different preference. If so pls update!)